### DISCUSSION TITLE
Acesso à dados com .NET, C#, Dapper e SQL Server

### DISCUSSION THREAD

**Discussion by PedroGasquez** [balta-io/forum#507] [Tira Dúvidas] [answered]
Fiz exatamente como nos videos mas na hora de printar a informação do banco de dados fica em branco o que pode ser?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @Stal91, poderia detalhar mais, com trechos de código e prints, todo o processo que ocorre, os códigos que escreveu e o que exatamente não está sendo exibido?

Informa também sua connection string e um print da tabela no banco com dados populados... assim podemos te ajudar melhor!

**Comment by PedroGasquez** [comment]
consegui resolver o problema era o banco que tinha sido gerado com erro
